amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/OasLikeOperationParser.scala

```diff
@@ -18,22 +18,22 @@ import amf.plugins.document.webapi.parser.spec.oas.{
 }
 import amf.plugins.domain.webapi.metamodel.OperationModel.Method
 import amf.plugins.domain.webapi.metamodel.{OperationModel, ResponseModel, WebApiModel}
-import amf.plugins.domain.webapi.models.{Operation, Request, Response}
+import amf.plugins.domain.webapi.models.security.SecurityRequirement
+import amf.plugins.domain.webapi.models.{Operation, Request, Response, security}
 import amf.validations.ParserSideValidations.{DuplicatedOperationId, InvalidStatusCode}
 import org.yaml.model._
 
 import scala.collection.mutable
 
-abstract class OasLikeOperationParser(entry: YMapEntry, adopt: Operation => Operation)(
-    implicit val ctx: OasLikeWebApiContext)
+abstract class OasLikeOperationParser(entry: YMapEntry, parentId: String)(implicit val ctx: OasLikeWebApiContext)
     extends SpecParserOps {
 
   protected def methodNode: AmfScalar = ScalarNode(entry.key).string()
 
   def parse(): Operation = {
     val operation: Operation = Operation(Annotations(entry))
-    operation.set(Method, methodNode) // add lexical info
-    adopt(operation)
+    operation.set(Method, methodNode, Annotations.inferred()) // add lexical info
```

Comment:
If `// add lexical info` is no longer required as it has already been done, please erase it.